supernovacontroller/sequential/i3c.py

```diff
@@ -885,14 +885,17 @@ def ccc_rstdaa(self, target_address):
 
         return self._process_response("ccc_rstdaa", responses)
 
-    def ccc_broadcast_enec(self):
+    def ccc_broadcast_enec(self, events: list):
         """
         Performs a broadcast ENEC (Enable Events Command) operation on the I3C bus.
 
         This method sends a broadcast command to enable events on all devices on the I3C bus.
         The operation's success status is checked, and it returns a tuple indicating whether the operation
         was successful along with the relevant data or error message.
 
+        Args:
+        events: A list containing events to be enabled.
```

Comment:
Please, mention that the type of element of the list belongs to the enum ENEC: ``ENEC.ENINT``, ``ENEC.ENCR``, ``ENEC.ENHJ``. For instance [ENEC.ENINT, ENEC.ENHJ]